src/components/ChartBar/ChartBar.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import ChartBarLegend from '../ChartBarLegend/ChartBarLegend.js'
+import {Bar} from 'react-chartjs-2';
+import axios from 'axios';
+
+class ChartBar extends React.Component{
+    state ={
+        chartMarket: [],
+        chartLabel: []
+    }
+    getChartInfo = async(url) => {
+      const {data} = await axios(url)
+      const market = data.total_volumes.map((el) => el[1])
+      const label = data.total_volumes.map(el => {
+        return new Date(el[0]).getDate()
+      })
+      this.setState({chartMarket: market, chartLabel: label})
+    }
+    componentDidMount(){
+      this.getChartInfo('https://api.coingecko.com/api/v3/coins/bitcoin/market_chart?vs_currency=usd&days=30&interval=daily');
```

Comment:
why pass the url as an argument? are you gonna invoke this function sometime with a different url?